### PR TITLE
terminus: gnome2 cleanup

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Removed/updated gnome2 references (Part of #39976).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by jtojnar** [comment]
https://github.com/NixOS/nixpkgs/pull/41174#discussion_r191239721 applies here as well. Hmm, we could really use `mkElectronDerivation` function to stop repeating ourselves.

**Comment by worldofpeace** [comment]
> we could really use mkElectronDerivation function to stop repeating ourselves.

Ahh music too my ears. They are way too many expressions that look just like this one.

File: pkgs/applications/misc/terminus/default.nix [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 , freetype, fontconfig, dbus, libXi, libXcursor, libXdamage, libXrandr
 , libXcomposite, libXext, libXfixes, libXrender, libX11, libXtst, libXScrnSaver
 , libxcb, makeWrapper, nodejs
-, GConf, nss, nspr, alsaLib, cups, expat, libudev, libpulseaudio }:
+, nss, nspr, alsaLib, cups, expat, libudev, libpulseaudio }:
 
 let
   libPath = stdenv.lib.makeLibraryPath [
-    stdenv.cc.cc gtk2 atk glib pango gdk_pixbuf cairo freetype fontconfig dbus
+    stdenv.cc.cc gnome3.gtk2 atk glib pango gdk_pixbuf cairo freetype fontconfig dbus
```
Comment (jtojnar) [inline pkgs/applications/misc/terminus/default.nix]: Use top-level gtk2.

**Comment by worldofpeace** [comment]
@xeji There's this one too. Next time I try to make similar changes it'll be one pr :+1:

**Comment by xeji** [comment]
@GrahamcOfBorg  build terminus

**Comment by xeji** [comment]
199MB disk size (this package alone, not the closure) for _a simple terminal program_, including a 74MB executable blob.  We cannot build it from source because it would be too complicated. 
And some people call that modern software :frowning_face:

**Pull request merged**